File: non_native/lib/view/components/custom_white_text.dart
import 'package:flutter/material.dart';

class CustomWhiteText extends StatelessWidget {
  final String text;
  final double size;

  const CustomWhiteText({
    Key? key,
    required this.text,
    this.size = 14.0,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Text(
      text,
      style: TextStyle(
        color: Colors.white,
        fontWeight: FontWeight.bold,
        fontSize: size,
      ),
    );
  }
}

File: native_server/app/src/main/java/com/example/myapp/ui/components/CustomWhiteText.kt
package com.example.myapp.ui.components

import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.text.TextStyle
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.TextUnit
import androidx.compose.ui.unit.sp

@Composable
fun CustomWhiteText(text: String, size: TextUnit = 14.sp) {
    Text(
        text = text,
        style = TextStyle(
            color = Color.White,
            fontWeight = FontWeight.Black,
            fontSize = size
        )
    )
}